cfn/configs/dev/us-east-1/config.yml

```
@@ -88,7 +88,7 @@ context:
           Command: ["CMD-SHELL", "curl -f http://127.0.0.1:5000/watchman || exit 1"]
           Timeout: 20
       environment: &base_env # below is also applied to celery containers
-        ALLOWED_HOSTS: "*"
+        ALLOWED_HOSTS: "127.0.0.1"
```

I think this should have both the local IP and the hostname, right?
```suggestion
        ALLOWED_HOSTS: 127.0.0.1,muckrock-dev.news-engineering.aws.wapo.pub
```
(and the same for prod)